pep-0678.rst

```diff
@@ -106,6 +106,19 @@ exception includes a note of the minimal failing example::
         +------------------------------------
 
 
+Non-goals
+---------
+``__note__`` is *not* intended to carry structured data.  If your note is for use by
+a program rather than display to a human, we recommend instead chosing a convention
+for an attribute like e.g. ``err._parse_errors = ...`` on the error or ``ExceptionGroup``
+(`discussion <https://discuss.python.org/t/accepting-pep-654-exception-groups-and-except/10813/26>`__,
+`discussion <https://bugs.python.org/issue46431>`__).
```

Comment:
This might be a case where a footnote is more appropriate? Two links that just say "discussion" are somewhat unhelpful to the reader to tell if I need to care about them / read the entire thread / etc.

Reply:
I second this